packages/frontend/amp/components/AdConsent.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { JsonScript } from './AMPJSON';
```

Comment:
Can we improve the naming here? Ideally so that the export is the same as the filename (as this is a single component file) and also to drop the 'AMP' prefix. Nothing particularly wrong with doing that, just we try and avoid it everywhere within the amp project as it creates noise/prefixes everywhere.

Reply:
👍